server/protocols/jmap-rfc-8621-integration-tests/postgres-jmap-rfc-8621-integration-tests/src/test/resources/lmtpserver.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
```

Comment:
Q: Does james-postgres support lmtp?

Reply:
I don't know. Just copy paste :)) Do you want me to remove it?

Reply:
Not sure, Because when developed Postgres feature, I didn't see anywhere lmtp.
We can remove it first, if CI fail, we can revert :P